octoprint_FriendlyBeeper/__init__.py

```diff
@@ -153,14 +156,21 @@ def get_assets(self):
 
     def get_api_commands(self):
         return dict(
-            beep_test=[],
+            beep_test=[
+                'beep_method',
+                'custom_tone',
+                'duration',
+                'end_time',
+                'frequency',
+                'start_time',
```

Comment:
this seems hard to maintain. If we add a new setting, this would have to be updated. And if any new api commands are added and they need to know settings, it would be similar. I feel like it would just be easier to send it as part of "settings", and have a function to validate settings so it's less duplicated code.

Reply:
I got an idea for that. Let me implement this later this week.

Reply:
Technically, it's already "later this week" :rofl: